x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -77,37 +79,40 @@ const UnstyledProcessEventDot = React.memo(
   ({
     className,
     position,
-    event,
+    node,
+    nodeID,
     projectionMatrix,
-    isProcessTerminated,
     timeAtRender,
   }: {
     /**
      * A `className` string provided by `styled`
      */
     className?: string;
     /**
-     * The positon of the process node, in 'world' coordinates.
+     * The positon of the graph node, in 'world' coordinates.
      */
     position: Vector2;
     /**
-     * An event which contains details about the process node.
+     * An event which contains details about the graph node.
      */
-    event: SafeResolverEvent;
+    node: ResolverNode;
     /**
-     * projectionMatrix which can be used to convert `position` to screen coordinates.
+     * The unique identifier for the node based on a datasource id
      */
-    projectionMatrix: Matrix3;
+    nodeID: string | undefined;
     /**
-     * Whether or not to show the process as terminated.
+     * projectionMatrix which can be used to convert `position` to screen coordinates.
      */
-    isProcessTerminated: boolean;
+    projectionMatrix: Matrix3;
 
     /**
      * The time (unix epoch) at render.
      */
     timeAtRender: number;
   }) => {
+    if (nodeID === undefined) {
```

Comment:
Instead of throwing this error, instead make `nodeID` required.

Reply:
Can you clarify as `nodeID` can be undefined? For now, I've moved the error up to where the `ProcessEventDot` is called. This error was here before as well based on the safe entity id